src/components/DisplayDeck.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+const VALUES= [ "A", "2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K"];
+const SUITS = ["♥", "♠", "♦", "♣"];
+const TOTAL_OF_CARDS = 21;
+
+const BuildDeck = () => {
+  return (VALUES.flatMap((value) => SUITS.flatMap(suit => ({ value: value, suit: suit})))
+  )
+}
+let deck = BuildDeck();
+
+const shuffledDeck = [...deck].sort(() => Math.random() - 0.5);
+const trickDeck = [...shuffledDeck].slice(0, TOTAL_OF_CARDS);
+
+const DisplayDeck = () => {
+  const [trickCards, setTrickCards] = useState(trickDeck);
+  return(
+    <>
+    <div className="cards" data-testid ="cards">
+      {trickCards.map(({value, suit},index) => {
+        return (
+          <div className ={`card card--${suit}`} key={index}>{value}{suit}</div>
```

Comment:
Uma sugestão aqui, evite usar `index` de um item como `key` na hora de renderizar um array no React, prefira usar um identificador único para aquele item independente da posição do item no array (nesse caso aqui, poderia ser algo como ```key={`${value}-${suit}`}```). O motivo é que o React usa esse `key` para tentar otimizar a renderização quando algo muda na tela (quando ele vai re-renderizar, ele tenta modificar a menor quantidade de coisas possíveis no DOM, de forma com que ele sabe que se um item só mudou de lugar com outro ele pode só mudar a ordem dos dois itens no DOM e depois alterar o conteúdo deles), então se um item muda de lugar mas a `key` dele muda junto (porque agora o index dele é outro) a gente perde em otimização.

Aqu item um pouco mais sobre o assunto:
- https://pt-br.reactjs.org/docs/lists-and-keys.html#keys
- https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9

Reply:
Ah eu utilizei o index, pq não ta conseguindo usar o suit ou value, pq se repetia, realmente com a interpolação faz muito mais sentido, pq cada um só tem 1 value com 1 suit